airflow/configuration.py

```diff
@@ -235,9 +229,28 @@ def getfloat(self, section, key):
         return float(self.get(section, key))
 
     def read(self, filenames):
-        ConfigParser.read(self, filenames)
+        super(AirflowConfigParser, self).read(filenames)
         self._validate()
 
+    def has_option(self, section, option):
+        try:
+            self.get(section, option)
+            return True
+        except AirflowConfigException:
+            return False
+
+    def remove_option(self, section, option):
+        """
+        Remove an option if it exists in config from a file or
+        default config. If both of config have the same option, this remove
+        the option in config loaded from a file at first and remove it from
+        default config at next call.
```

Comment:
I think a clearer api here would be to have this method be `configuration.remove_option('core', 'something', remove_default=True)` - rather than having to call it twice.

Reply:
Changed the api to what you suggested.